docs/src/getting-started/checkpoints.rst

```diff
@@ -47,7 +47,8 @@ a file path.
     mtt export https://my.url.com/model.ckpt --output model.pt
 
 Downloading private HuggingFace models is also supported, by specifying the
-corresponding API token with the ``--huggingface_api_token`` flag.
+corresponding API token with the ``--huggingface_api_token`` flag or the
+``HUGGINGFACE_METATRAIN_TOKEN`` environment variable.
```

Comment:
Why HUGGINGFACE_METATRAIN_TOKEN and not just the HF_TOKEN, as official docs suggest?

https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables